data_handler.py

```diff
@@ -1,12 +1,13 @@
+import base64
+import datetime
+import io
+import time
+from datetime import timedelta
 from functools import reduce
 from typing import List
-import pandas as pd
-import time
+
 import numpy as np
-from datetime import timedelta
-import datetime
-import io
-import base64
+import pandas as pd
 
 
 class DataFilterer:
```

Comment:
Class name does not match file name

Reply:
Fixed in 4189132. Left the name as `data_handling` as the file has two classes, with that name a descriptor of their grouping.